client/src/components/Nav/style.css

```diff
@@ -1,15 +1,7 @@
 .nav-wrapper {
-    background-color: aqua;
+    background-color: #9b74c2;
```

Comment:
extract this hex color out to a css variable and use it here. This hex color isnt descriptive at all? is it blue? is it green? black? put it as a variable.